xwiki-platform-core/xwiki-platform-xml/xwiki-platform-xml-script/src/test/java/org/xwiki/xml/script/XMLScriptServiceTest.java

```diff
@@ -25,333 +25,381 @@
 
 import javax.xml.transform.stream.StreamSource;
 
-import org.junit.Assert;
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
 import org.w3c.dom.Document;
 import org.w3c.dom.Element;
 import org.w3c.dom.ls.LSInput;
-import org.xwiki.script.service.ScriptService;
-import org.xwiki.test.jmock.AbstractComponentTestCase;
-import org.xwiki.xml.script.XMLScriptService;
+import org.xwiki.test.junit5.mockito.ComponentTest;
+import org.xwiki.test.junit5.mockito.InjectMockComponents;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
 
 /**
  * Unit tests for {@link org.xwiki.xml.script.XMLScriptService}.
- * 
+ *
  * @version $Id$
  * @since 2.7RC1
  */
-public class XMLScriptServiceTest extends AbstractComponentTestCase
+@ComponentTest
+public class XMLScriptServiceTest
 {
+    @InjectMockComponents
     private XMLScriptService xml;
 
-    @Override
-    public void setUp() throws Exception
-    {
-        this.xml = (XMLScriptService) getComponentManager().getInstance(ScriptService.class, "xml");
-    }
-
-
-
     @Test
-    public void testGetDomDocument()
+    void testGetDomDocument()
```

Comment:
Note that we don't prefix test methods with `test`.  This one should be named `createDOMDocument()`. what I do when converting test, is usually to just remove the `test` prefix.

Rule: https://dev.xwiki.org/xwiki/bin/view/Community/Testing/JavaUnitTesting/#HBestpractices